LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+Copyright (c) 2020 CyberWay and its contributors.  All rights reserved.
```

Comment:
I suggest to write `2019-2020` as well as in https://github.com/cyberway/commun.contracts/pull/604/files#diff-9879d6db96fd29134fc802214163b95aR1